spec/data-model.md

```diff
@@ -101,16 +101,16 @@ The `Literal` and `VariableRef` correspond to the the _literal_ and _variable_ s
 When they are used as the `body` of an `Expression`,
 they represent _expression_ values with no _annotation_.
 
-An _unquoted_ value is represented by a `Literal` with `quoted: false`,
-while a _quoted_ value would have `quoted: true`.
+Both _quoted_ and _unquoted_ values are represented by `Literal`,
+as the use or lack of quotation is a presentation detail
+which has no effect on the meaning of the _literal_.
 The `value` of `Literal` is the "cooked" value (i.e. escape sequences are processed).
```

Comment:
I would avoid the jargon-ish "cooked" here and I would note the non-inclusion of the quotes (where they exist and as you did for reserved's sigils elsewhere)

```suggestion
The `value` of `Literal` does not include surrounding quotes (where present)
and replaces `quoted-escape` sequences with the unescaped character.
```

Reply:
I'd like for this to be considered in a separate issue or PR, for two reasons:
1. The data model uses the "raw" and "cooked" terms also with respect to `Text` and `Reserved`, which ought to be updated simultaneously. I would rather keep that outside the scope of this rather focused PR.
2. At the moment, the data model is described and explained via equivalences with the MF2 syntax. If there is a desire to describe it as an explicit result of parsing the syntax, as suggested by the term "replaces" here, that's a much bigger change that ought to be accompanied by some additional documentation.

Reply:
If "raw" and "cooked" are terms, define them has terms (in a terminology section that needs to be added) and link them.

A different way of saying this would be to go more Unicode-like:

```suggestion
The `value` of `Literal` is the code point sequence contained by the _literal_,
with external syntax (such as quotes) removed
and escape sequences resolved to the characters that they represent.
```

This would apply to any representation, not just MF2. For example, a JS string would replace `\u20ac` notation with `€` in the `Literal`.

> If there is a desire to describe it as an explicit result of parsing the syntax, as suggested by the term "replaces" here, that's a much bigger change that ought to be accompanied by some additional documentation.

I think we should stipulate that the data model representation can round-trip any MF2 string without loss of information,  although doing so would canonicalize the representation, such as syntax (non-literal) whitespace and the presence or absence of quotes around literals such that the resulting round-trip string might not be a character-by-character match to the original.